q2.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <pthread.h>
+#include <math.h>
+
+#define tam 10
```

Comment:
```suggestion
#define TAM 10
```
Por padrão, os defines são com letras maiúsculas para diferenciar das variáveis normais